docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -290,15 +290,15 @@ Scenario: Open Epam main page
 Meta:
   @severity 1
   @layout desktop
-Given I am on a page with the URL 'https://www.epam.com/'
+Given I am on page with URL `https://www.epam.com/`
```

Comment:
We need to leave at least one test per step for deprecated syntax

Reply:
Accessibility.story:7 - Given I am on a page with the URL
WebCookieSteps.story:5 - Given I am on the main application page